libs/media-player/src/lib/timecode-formatted.tsx

```diff
@@ -0,0 +1,29 @@
+interface MediaCurrentTimeFormattedProps {
+    mediaCurrentTime: number;
+}
+
+export const MediaCurrentTimeFormatted = ({
```

Comment:
Definitely, though `currentTime` is the property to get and set the video player pointer in the native HTML player, so I was trying to be consistent with that rather than inventing our own parallel language for their API.